testing/client.related.test.js

```diff
@@ -1,83 +1,116 @@
-/**
- * @jest-environment jsdom
- */
-
-import {cleanup, fireEvent, render, screen} from '@testing-library/react';
+import { render, screen, waitFor, fireEvent } from '@testing-library/react';
```

Comment:
@sm-fm here is where the important changes are for my tests.

Reply:
Beautiful